### PR TITLE
Fix return description in hover content

### PR DESCRIPTION
**Pull request by nadeeshaan**
## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by malinthar — APPROVED** [review]

File: language-server/modules/langserver-core/src/test/resources/hover/configs/hover_remote_method_decl2.json [hunk 1]
```diff
@@ -1,21 +1 @@
-{
```
Comment (hasithaa) [inline language-server/modules/langserver-core/src/test/resources/hover/configs/hover_remote_method_decl2.json:1]: Are these changes OK? We used to format those previously.

**Comment by hasithaa** [comment]
@nadeeshaan What are the other places we show this, in VSCode? Hope we have covered all the places.

File: language-server/modules/langserver-core/src/test/resources/hover/configs/hover_remote_method_decl2.json (hunk 1 above)
Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/test/resources/hover/configs/hover_remote_method_decl2.json:1]: This is rectified

**Comment by nadeeshaan** [comment]
> @nadeeshaan What are the other places we show this, in VSCode? Hope we have covered all the places.

We have to change the completion documentations' `Returns` title with `Return` title. Which will be sent in a separate commit

**Review by hasithaa — APPROVED** [review]

**Pull request merged**